packages/edit-site/src/components/sidebar-edit-mode/page-panels/hooks.js

```diff
@@ -4,12 +4,18 @@
 import { useSelect } from '@wordpress/data';
 import { useMemo } from '@wordpress/element';
 import { store as coreStore } from '@wordpress/core-data';
+import { parse } from '@wordpress/blocks';
 
 /**
  * Internal dependencies
  */
 import { store as editSiteStore } from '../../../store';
-import { TEMPLATE_POST_TYPE } from '../../../utils/constants';
+import {
```

Comment:
Is there a specific reason for having this code in `page-panels` and not in `template-panel`?

Reply:
Historical reasons I think

Reply:
I think it's better suited to the panel is used. WDYT?

Reply:
Yeah I'll move it